tests/executor/test_executor.py

```diff
@@ -188,3 +198,146 @@ def test_clear(tmpfile):
         status = f.post(on='/status', return_results=True)[0]
         assert int(status.tags['total_docs']) == 0
         assert int(status.tags['index_size']) == 0
+
+
+@patch.dict(os.environ, {'JINA_AUTH_TOKEN': ''})
+def test_remote_storage(tmpfile):
+    os.environ['JINA_AUTH_TOKEN'] = 'ed17d158d95d3f53f60eed445d783c80'
+    clear_hubble()
+
+    docs = gen_docs(N)
+    f = Flow().add(
+        uses=AnnLiteIndexer,
+        uses_with={
+            'n_dim': D,
+        },
+        workspace=tmpfile,
+        shards=1,
+    )
+    with f:
+        f.post(on='/index', inputs=docs)
+        time.sleep(2)
+        f.post(on='/backup', parameters={'target_name': 'backup_docs'})
+        time.sleep(2)
+
+    f = Flow().add(
+        uses=AnnLiteIndexer,
+        uses_with={'n_dim': D, 'restore_key': 'backup_docs'},
+        workspace=tmpfile,
```

Comment:
this will restore the indexer from `workspace` instead of Hubble

Reply:
no, once the `restore_key` is specified, annlite indexer will restore from hubble and will not check local file. If not, it will check local file and restore from workspace. you could refer to `__init()__` in `AnnLiteIndexer`.